src/cmd/team.go

```diff
@@ -103,46 +103,6 @@ opslevel create contact --type=email my-team team@example.com "Mailing List"`,
 	}
 )
 
-var createTeamTagCmd = &cobra.Command{
```

Comment:
need to do the deprecation flow for these `opslevel create team tag` too

Reply:
added the deprecation flow to team.go too